assets/css/utilites.css

```diff
@@ -0,0 +1,164 @@
+.container {
+  max-width: 1300px;
+  margin: 0 auto;
+  overflow: auto;
+  padding: 0 40px;
+}
+
+.card {
+  background-color: #fff;
+  border-radius: 24px;
+  box-shadow: 0 3px 10px rgba(0, 0, 0, 0.2);
+  padding: 20px;
+  margin: 10px;
+}
+
+.btn {
+  display: inline-block;
+  padding: 10px 10px;
+  cursor: pointer;
+  background: var(--primary-color);
+  color: #fff;
+  border: none;
+  border-radius: 5px;
+}
+
+.btn:hover {
+  transform: scale(0.9);
+}
+
+.btn-hover:hover {
+  background: #6070ff;
+  color: #fff;
+}
+
+.btn-hover-2:hover {
+  border: 1px solid #fff;
+  background: #6070ff;
+  color: #fff;
+}
```

Comment:
- [ ] Well done for implementing the button interactions 🥇🥇 However, I noticed that you missed the `box-shadow` property for `hover` which is `box-shadow: 0px 8px 16px rgba(64, 83, 252, 0.24);` Please add this ✨